plugins/single_plugins/place.cpp

```diff
@@ -6,8 +6,14 @@
 
 class wayfire_place_window : public wf::plugin_interface_t
 {
-    wf::signal_callback_t created_cb;
-    wf::signal_callback_t workarea_changed_cb;
+    wf::signal_connection_t created_cb = [=] (wf::signal_data_t *data)
+    {
+        do_created(data);
```

Comment:
You should put the whole code inside the lambda, this is how we usually do it.

Reply:
Oh ok, will do. I was looking at the window-rules plugin for comparison, but I guess that was done that way to reuse the same function.